cpp/tests/interop/nanoarrow_utils.hpp

```diff
@@ -17,6 +17,7 @@
 #pragma once
```

Comment:
Just a note. This file is included by the interop benchmark source too so I'm moving it to the `cpp/include/cudf_tests` directory here: https://github.com/rapidsai/cudf/pull/18163